lib/room.rb

```diff
@@ -1,12 +1,25 @@
-class Rom
+class Room
+  attr_reader :category,
+              :length,
+              :width,
+              :is_painted
```

Comment:
We don't need a method `is_painted`; we only want the `is_painted?` method below.